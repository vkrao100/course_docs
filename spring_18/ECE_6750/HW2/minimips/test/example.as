load_map:      addi   $a0, $zero, 360        # Base address for array v
               addi   $a1, $zero, 30        
                                         
               add    $t0, $a0, $zero        
               add    $t1, $a1, $zero        
               add    $t2, $zero, $zero      

load_no_map:   addi   $0, $0, 360            # Base address for array v
               addi   $1, $0, 30        
                                         
               add    $0, $0, $0        
               add    $1, $1, $0        
               add    $2, $0, $0      
