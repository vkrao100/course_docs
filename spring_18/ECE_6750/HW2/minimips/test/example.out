00100000000001000000000101101000
00100000000001010000000000011110
00000000100000000100000000100000
00000000101000000100100000100000
00000000000000000101000000100000
00100000000000000000000101101000
00100000000000010000000000011110
00000000000000000000000000100000
00000000001000000000100000100000
00000000000000000001000000100000
